### ISSUE TITLE
Red Cross beta testers are wary of using their social media accounts to login to allReady and would prefer a 'direct' login option

### ISSUE THREAD

**Issue by OhMcGoo** [HTBox/allReady#330]

**Comment by dpaquette** [comment]
This should be available already. Was fixed as part of #319

**Comment by dpaquette** [comment]
See also #313

**Comment by BillWagner** [comment]
Depends on which host. It should be deployed on -d. @tonysurma would know if it's deployed on -mvp

**Comment by tonysurma** [comment]
Later today

**Comment by tonysurma** [comment]
deployed

**Comment by tonysurma** [comment]
@dpaquette we may need to reopen this or make a new issue around password rules, I am a kinda smart tech guy and I can't figure out how to make a password that works for it and @OhMcGoo said others were having confusion as well.  What are the rules currently?

**Comment by tonysurma** [comment]
I think I just successfully set a password but then I got this error

![snip_20151112232022](https://cloud.githubusercontent.com/assets/5067595/11139366/078ad8f4-8994-11e5-8ce6-7762c55ced95.png)

**Comment by OhMcGoo** [comment]
Still learning to get my comments in the RIGHT issue:

Let me try again.

The direct login feature isn’t quite there. My password was rejected on my first attempt 

![create account - 1st attempt](https://cloud.githubusercontent.com/assets/4147733/11139335/acce4ed2-8993-11e5-8830-7c92debfecdd.png)

NOTE: This phrasing is confusing:  "Passwords must have at least one non letter and non digit character." A non letter non digit is a symbol, right?

My second attempt brought about quite an astonishing display of characters!

![create account - 2nd attempt](https://cloud.githubusercontent.com/assets/4147733/11139336/ace4c644-8993-11e5-8aec-eae09d7f810c.png)

**Comment by dpaquette** [comment]
pinging @MisterJames 
Is something not configured for email notifications to work on this server?

**Comment by dpaquette** [comment]
Maybe re-open this as the queue storage issue is still outstanding

**Comment by tonysurma** [comment]
@MisterJames said he will look at queue later today.

**Comment by MisterJames** [comment]
@tonysurma this one should close with the merge of #343. Once that's confirmed, feel free to close this out.

**Comment by tonysurma** [comment]
@MisterJames yep will close once deployed and confirmed

**Comment by tonysurma** [comment]
between an extra config fix and #343 per @MisterJames I am pretty sure this will be closed out today.

**Comment by tonysurma** [comment]
Reopening as there are no webjobs to process the queues :cry: showing up in azure websites cc @MisterJames @dpaquette

**Comment by tonysurma** [comment]
For now this works with manual deployment to -mvp.  So closing this but opening an issue in UAT to get it working with kudu deployment

**Comment by tonysurma** [comment]
verified working
